Examples/Demo_Base.strings

```diff
@@ -7,3 +7,6 @@
 "mismatch %@ types %d" = "mismatch %@ types %d";
 
 "bad position %d" = "bad position %d";
+
+"duplicate key" = "duplicate key";
+"duplicate key" = "duplicate key";
```

Comment:
[nit] Should we also consider a test where the keys are duplicated but the values are not?

Reply:
I considered doing this, but I don't think it would catch any cases that wouldn't be caught anyway. At best it would be a slightly better error message.